src/frontend/src/pages/CalendarPage/DesignReviewCreateModal.tsx

```diff
@@ -71,13 +72,16 @@ export const DesignReviewCreateModal: React.FC<DesignReviewCreateModalProps> = (
   const [requiredMembers, setRequiredMembers] = useState([].map(userToAutocompleteOption));
   const [optionalMembers, setOptionalMembers] = useState([].map(userToAutocompleteOption));
   const { isLoading: allUsersIsLoading, isError: allUsersIsError, error: allUsersError, data: users } = useAllUsers();
+
   const {
     isLoading: allWorkPackagesIsLoading,
     isError: allWorkPackagesIsError,
     error: allWorkPackagesError,
     data: allWorkPackages
   } = useAllWorkPackages();
 
+  const { data: allProjects } = useAllProjects();
```

Comment:
i really dont like that were getting all the projects here... i suppose this is the only real way to do it for now, we may want to add a project preview to a work package that includes ig the team type or smth idrk